TAF_EasyRest/TestFramework/PageComponents/Client/RestaurantMenuComponents/OrderSummaryDetailsPageComponent.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace TestFramework.PageComponents
+{
+    public class OrderSummaryDetailsPageComponent
+    {
+        private IWebDriver driver;
+        private int index;
+
+        public OrderSummaryDetailsPageComponent(IWebDriver driver, int index)
+        {
+            this.driver = driver;
+            this.index = index;
+        }
+
+        #region Elements
+        private IWebElement _currentElement => driver.FindElement(By.XPath($"(//td[@class='MuiTableCell-root-3326 MuiTableCell-body-3328 MuiTableCell-paddingCheckbox-3332']//parent::tr)[{index}]"));
```

Comment:
I can't find these elements. Please check.

Reply:
You can't use class names with numbers like this td[@Class='MuiTableCell-root-3018'] because these numbers are auto-generated during each launch. Try to use instead: td[contains(@Class, 'MuiTableCell-root')]